lib/spack/docs/developer_guide.rst

```diff
@@ -409,7 +426,10 @@ or a file:
 
    $ spack python ~/test_fetching.py
 
-just like you would with the normal ``python`` command.
+just like you would with the normal ``python`` command. Note that for both
+commands and files, the default will always be the python interpreter,
```

Comment:
With new edits, `-i ipython` will through an error so I think comment can be updated accordingly.

Reply:
My bad, totally forgot about updating the docs! I'll do that now.